features/bevragen/persoon/adressering/adres-regels/in-onderzoek.feature

```diff
@@ -0,0 +1,116 @@
+    | inOnderzoek.land                                           | <land io>        |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.type        | Datum            |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.datum       | 2002-07-01       |
+    | inOnderzoek.datumIngangOnderzoekVerblijfplaats.langFormaat | 1 juli 2002      |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | adresregel1 io | adresregel2 io | adresregel3 io | land io | type                               |
+    | 080000                  | true           | true           | true           | true    | hele categorie verblijfplaats      |
+    | 080900                  |                | true           |                |         | hele groep gemeente                |
+    | 080910                  |                | true           |                |         | gemeente van inschrijving          |
+    | 081000                  |                |                |                |         | hele groep adreshouding            |
+    | 081010                  |                |                |                |         | functie adres                      |
+    | 081030                  |                |                |                |         | datum aanvang adreshouding         |
+    | 081100                  | true           | true           |                |         | hele groep adres                   |
+    | 081110                  | true           |                |                |         | straatnaam                         |
+    | 081115                  |                |                |                |         | naam openbare ruimte               |
+    | 081120                  | true           |                |                |         | huisnummer                         |
+    | 081130                  | true           |                |                |         | huisletter                         |
+    | 081140                  | true           |                |                |         | huinummertoevoeging                |
+    | 081150                  | true           |                |                |         | aanduiding bij huisnummer          |
+    | 081160                  |                | true           |                |         | postcode                           |
+    | 081170                  |                | true           |                |         | woonplaatsnaam                     |
+    | 081180                  |                |                |                |         | identificatiecode verblijfplaats   |
+    | 081190                  |                |                |                |         | identificatiecode nummeraanduiding |
+    | 088500                  |                |                |                |         | hele groep geldigheid              |
+    | 088510                  |                |                |                |         | datum ingang geldigheid            |
+
```

Comment:
ik mis hier onderzoek van groep 12 en 13. Bijvoorbeeld persoon heeft nu adres in NL, maar onderzocht wordt of de persoon naar het buitenland is vertrokken. Als dan inOnderzoek staat op 081300, zou ik de adresregels wel op inOnderzoek zetten. Zo staat dat ook in [/features/in onderzoek.xlsx](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/develop/features/in%20onderzoek.xlsx)

Reply:
aanduiding in onderzoek voor groep 12 en 13 zijn toegevoegd